README.md

```diff
@@ -21,8 +21,8 @@
 
 ## What is tldr-pages?
 
-The **tldr-pages** project is a collection of community-maintained help pages
-for command-line tools, that aims to be a simpler, more approachable complement
+The **tldr-pages** project is the biggest collection of community-maintained help pages
```

Comment:
This feels wrong. Objectivity is very important to us in our pages, so I think this should extend to our README as well.

It feels far too much like a marketing ploy that would be used by a for-profit business rather than an open-source project.

What is to say if there's some other project out there that is larger than us? Would this then become inaccurate? How can we verify the truth of this claim?

We have 47.5K stars here on GitHub, if people are looking at the README then they have likely seen that number, which can speak for itself.

Example other open-source project websites for reference:

- https://www.gimp.org/ - "The Free & Open Source Image Editor"
- https://inkscape.org/ - "A powerful, free design tool"
- https://react.dev/ - "The library for web and native user interfaces"
- https://www.blender.org/ - "blender.org - Home of the Blender project - Free and Open 3D Creation Software"